### DISCUSSION TITLE
Fractional scaling support

### DISCUSSION THREAD

**Discussion by cr1ogen** [qtile/qtile#4034] [Ideas]
Wayland now support  fractional scaling https://www.phoronix.com/news/Wayland-Fractional-Scale-Ready 
Could this be implemented in qtile?

**Comment by m-col** [comment]
For sure. I see the wlroots guys are putting together some library code that we can then work into pywlroots and them qtile: https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/3511

I've added the feature to the wayland to do list: https://github.com/qtile/qtile/discussions/2409
